### DISCUSSION TITLE
Wechselrechter erkannt, keine Daten / Inverter recognised, no data

### DISCUSSION THREAD

**Discussion by carpodaster** [tbnobody/OpenDTU#1132] [Q&A]
(for :gb: / :us:, see below)

Hi! 

Hab mir gestern meine [blinkyparts DTU zusammengelötet](https://binary-kitchen.github.io/SolderingTutorial/OpenDTU_Breakout/manual/OpenDTU_Breakout_de.pdf) (mein erstes Mal mit Lötkölben an Elektronikbauteilen). Einheit bootet, ist in meinem WLAN und erkennt nach Eingabe der S/N auch meinen WR. Allerdings bekommt er keine Daten (siehe Bild unten). 

LEDs und Display machen auch nichts, aber ich weiß nicht ob die nicht erst was anzeigen sollen, wenn die Verbindung steht.

Hab ich mich verlötet? Die Tatsache, dass er meinen WR per Funk erkennt, lässt mich denken, dass die Funkkommunikation zumindest i.O. ist.

Ich hab hier von einer seriellen Konsole gelesen, aber das einzige, was ich hab, ist die "Konsole" im Webinterface. Die zeigt mir gar nichts.

MQTT (siehe unten) gibt mir auch nichts, das mir bei der Fehlersuche weiter hilft.

Hat jemand irgendwelche Hinweise für mich, wie ich weitermachen kann? Danke!

### English

I finished soldering my [blinkyparts DTU zusammengelötet](https://binary-kitchen.github.io/SolderingTutorial/OpenDTU_Breakout/manual/OpenDTU_Breakout_en.pdf) yesterday. It boots up, I successfully integrated it into my home wifi, and it also recognised my inverter after I entered its serial number. However, it won't receive any data (see screenshot below).

LEDs and display also don't do anything but I don't know if they are only supposed to fire up once the connection to an inverter is established.

Did I do something wrong? The fact that the DTU recognises my inverter let me think that the radio bit at least is working. 

I've seen people referring to a "serial log". I can only see the "console" on the web interface and it doesn't show anything. 

MQTT (see below) also doesn't give me anything useful to debug. 

Does anyone have pointers on how to proceed? Thanks!

![grafik](https://github.com/tbnobody/OpenDTU/assets/164400/a150e7de-e390-4ac7-a31c-532dca91e8c9)

```
$ mosquitto_sub -h tinkerboard -v -t openDTU/#
openDTU/dtu/status online
openDTU/dtu/uptime 644
openDTU/dtu/ip 192.168.178.198
openDTU/dtu/hostname OpenDTU-D79CA8
openDTU/dtu/rssi -68
openDTU/dtu/bssid 2C:91:AB:25:27:A0
openDTU/ac/power 0.0
openDTU/ac/yieldtotal 0.000
openDTU/ac/yieldday 0
openDTU/ac/is_valid 1
openDTU/dc/power 0.0
openDTU/dc/irradiation 0.000
openDTU/dc/is_valid 1
openDTU/SERIALNO/name HM-600 West
openDTU/SERIALNO/status/reachable 1
openDTU/SERIALNO/status/producing 0
openDTU/SERIALNO/status/last_update 0

**Comment by tbnobody** [comment]
> Die Tatsache, dass er meinen WR per Funk erkennt, lässt mich denken, dass die Funkkommunikation zumindest i.O. ist.

Woran machst  du fest eine eine Funkkommunikation vorhanden ist? Was sagt denn Info --> System --> Funkmodulinformation? Wird dort eine Verbindung festgestellt?

Das  Display sollte zumindest  in den ersten Minuten nach dem Einschalten der DTU aktiv sein.

**Comment by carpodaster** [comment]
> Woran machst du fest eine eine Funkkommunikation vorhanden ist? 

Ich hab die echte S/N eingegeben und er listet etwas in der Live-Ansicht. Wenn ich eine Bogus-S/N eingegeben habe, hat er den WR in der Liste erst gar nicht aufgeführt.

> Info --> System --> Funkmodulinformation

Aber: nachdem ich das hier :arrow_down: gesehen habe, vermute ich, dass er den [WR Typ eher aus einer hartkodierten S/N Map entnimmt](https://github.com/tbnobody/OpenDTU/blob/9b0d2ff25f48fd39155894351d4069fba5900efa/lib/Hoymiles/src/inverters/HM_2CH.cpp#L42-L65)?

![grafik](https://github.com/tbnobody/OpenDTU/assets/164400/c949e599-0253-4dd6-a7a0-0cf78631dc7d)

Also ja, ich mach das Ding nochmal wieder auf und überprüfe die Verbindungen :electric_plug:   Danke schonmal!

**Comment by fouille** [comment]
I've same problem : https://github.com/tbnobody/OpenDTU/discussions/1202
